src/azul/transformer.py

```diff
@@ -300,6 +300,43 @@ def get(self):
         return thaw(super().get())
 
 
+class FrequencySetAccumulator(Accumulator):
+    """
+    An accumulator that accepts any number of values and returns a list that has
```

Comment:
The ordering property should remain undocumented since it is currently not used, unlikely to ever be used and therefore not worth constraining future implementations by.